apis/user-java/README.md

```diff
@@ -0,0 +1,18 @@
+# Swagger generated server
+
+Spring Boot Server 
+
+
+## Overview  
+This server was generated by the [swagger-codegen](https://github.com/swagger-api/swagger-codegen) project.  
+By using the [OpenAPI-Spec](https://github.com/swagger-api/swagger-core), you can easily generate a server stub.  
+This is an example of building a swagger-enabled server in Java using the SpringBoot framework.  
+
+The underlying library integrating swagger to SpringBoot is [springfox](https://github.com/springfox/springfox)  
+
+Start your server as an simple java application  
+
+You can view the api documentation in swagger-ui by pointing to  
+http://localhost:8080/  
```

Comment:
Can you please add the current URLS which actually serve the dump of the swagger JSON and the one which serves the html page?